### PR TITLE
fix env var not found: DATABASE_URL

### PR DESCRIPTION
**Pull request by andriisoldatenko**
## Description

> Describe the purpose of this pull request.

## 🎟 Issue(s)

Resolves astronomer/issues#1858

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

## 📋 Checklist

- [ ] Rebased from the main (or release if patching) branch (before testing)
- [ ] Ran `yarn lint && yarn test` before taking out of draft
- [ ] Added/updated applicable tests
- [ ] Tested against [Astro-UI](https://github.com/astronomer/astro-ui/) and [Astro-CLI](https://github.com/astronomer/astro-cli/) clients.
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

**Comment by adam2k** [comment]
I didn't realize that you already opened this PR. I'll close my PR and tag the issue in this PR https://github.com/astronomer/issues/issues/1858

**Review by adam2k — APPROVED** [review]
Looks good to me!  🎉

File: README.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ Houston leverages [Prisma](https://www.prisma.io/docs/) as a [GraphQL](https://g
 ## Configuration
 
 Houston can be configured via YAML files under `./config`, and can be overridden via environment variables, which are defined in `/config/custom-environment-variables.yaml`. To add or override any variables locally, create a `.env` file in the project directory.
+> Note: If you develop locally with prisma2, you need to run:
+>```bash
+>cp ./prisma/.env.example ./prisma/.env 
+> than you can see message in logs:
```
Comment (adam2k) [inline README.md:21]: ``Suggestion
then you will see a message like this in the logs:
```

**Comment by andriisoldatenko** [comment]
@ryanahamilton PTAL

**Review by ryanahamilton — APPROVED** [review]
Worked for me—thank you!

**Pull request merged**